mutmut/__main__.py

```diff
@@ -885,14 +885,14 @@ def print_stats(source_file_mutation_data_by_path, force_output=False):
     print_status(f'{(s.total - s.not_checked)}/{s.total}  🎉 {s.killed} 🫥 {s.no_tests}  ⏰ {s.timeout}  🤔 {s.suspicious}  🙁 {s.survived}  🔇 {s.skipped}', force_output=force_output)
 
 
-def run_forced_fail(runner):
+def run_forced_fail_test(runner):
     os.environ['MUTANT_UNDER_TEST'] = 'fail'
     with CatchOutput(spinner_title='Running forced fail test') as catcher:
         try:
             if runner.run_forced_fail() == 0:
                 catcher.dump_output()
-                print("FAILED")
-                os._exit(1)
+                print("Error: Unable to force a test failure during the forced fail test")
+                raise SystemExit(1)
```

Comment:
I would suggest "FAILED: Unable to force test failures". To be consistent, and for brevity.

Why did you change the exit? I use os._exit() for fast exit, so we don't spend time deallocating at the end of the process.

Reply:
I have update the error message.

I switched to `SystemExit` because it is much easier to write a test for. Re the performance thing, I think this is something that will only happen once, early on in a run, if something has gone wrong, so some milliseconds taken occasionally doesn't seem like a problem.

Reply:
Not sure why `test_run_forced_fail_test_with_all_tests_passing` ia passing locally, but not in GitHub Actions. I'll come back to that when I next have some time.

Reply:
All the unit tests I added are failing in GitHub Actions because the output emitted by `run_forced_fail_test` is not making it to `stdout`.

I have tried adding some sleeps to rule out a race condition and changing the runner to MacOS in case that was a factor. No banana.